jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/grid/editor/DataGridEditor.java

```diff
@@ -64,8 +64,7 @@ public interface DataGridEditor<T> extends Editor<T> {
      * Sets a function that returns the column editor component. To bound an editor component
      * to the passed entity's property, the function implementation should use
      * {@link EditComponentGenerationContext#getValueSourceProvider} to obtain an instance
-     * of {@link ValueSource}. To find the column it's assumed that the column key is equal
-     * to the property.
```

Comment:
Perhaps this description in JavaDocs makes the task irrelevant, doesn't it?

Reply:
Seems that no one cares what JavaDoc says 😃